source/systems/research.md

```diff
@@ -206,97 +224,129 @@ You can only choose one of these!
 | Full Stock | Capacity | Mine level 5; 64 racks | Increases the number of items you can set as a minimum stock in the Warehouse, Restaurant, and Bakery GUIs by 200% |
 | RTM (Read the Manual) | Recipe book | Sawmill at least level 4; 256 paper | Increases the max amount of recipes stored in each hut by 200% |
-| Loaded | Deep Pockets | Library level 5; 64 emeralds | Adds 18 slots to colonists' inventories |
+| Loaded | Deep Pockets | Library level 5; 128 emeralds | Adds 18 slots to colonists' inventories |
 | What Is This Speed? | Heavy Machinery | 1024 redstone dust | Increases colonists' block breaking speed by 100% |
-| Steel Bracing | Reinforced | Blacksmith's Hut level 5; 64 diamonds | Increases colonists' tools' durability by 50% |
-| Seems Automatic | Tools | Blacksmith's Hut level 5; 512 iron ingots | Increases colonists' block placing speed by 100% |
-| Amazing Veins | Rich Veins | Blacksmith's Hut level 5; 64 gold ore | Increases the number of ores the Miner produces by 100% |
 
 #### Sixth Column
 
-You can only choose one of these!
+You can only have one sixth column research unlocked at a time!
 
 | Name | Prerequisites | Requirements | Effects |
 | ---- | ------------- | ------------ | ------- |
 | Magic Compost | Fertilizer | 2048 wheat seeds | Increases Farmers' output by 200% |
-| Rainman | RTM | 27 buckets of salmon | Workers work during rain and storms |
-| Heavily Loaded | Loaded | 128 emeralds | Adds 27 slots to colonists' inventories |
-| Lightning | What Is This Speed? | 2048 redstone dust | Increases colonists' block breaking speed by 200% |
 | Diamond Coated | Steel Bracing | 128 diamonds | Increases colonists' tools' durabilty by 90% |
 | Madness! | Seems Automatic | 1024 iron ingots | Increases colonists' block placing speed by 200% |
-| Motherlode | Amazing Veins | 64 diamond ore | Increases the number of ores the Miner produces by 200% |
+| Motherlode | Amazing Veins | 64 diamond ore | Miner produces 200% more ores |
+| Rainman | RTM | 27 buckets of salmon | Workers work during rain and storms |
+| Heavily Loaded | Loaded | 256 emeralds | Adds 27 slots to colonists' inventories. **This research can not be undone.** |
+| Lightning | What Is This Speed? | 2048 redstone dust | Increases colonists' block breaking speed by 200% |
+
+### Unlockables
+<br>
+<img src="../../assets/images/gui/university/unlockables.png" class="img-fluid mx-auto" alt="University Unlockables GUI Page">
+<br>
+
+Unlockables are a special research tree, consisting of research that other colonists discover as the colony grows and progresses. This branch will only display once at least one research is available, and research on this branch will complete instantly.
+
+#### First Column
+
+| Name         | Requirements             | Effects |
+| ------------ | ------------------------ | ------- |
+| String Mesh  | Sifter at least level 1 | Fletcher learns the String Mesh recipe |
```

Comment:
links to the meshes, sifter, and fletcher here (for the other unlockables as well)

Reply:
sifter, or sifter's hut?